src/utils/includes.h

```diff
@@ -49,11 +49,13 @@
 #include <zephyr/posix/sys/select.h>
 #include <zephyr/posix/sys/socket.h>
 #include <zephyr/posix/unistd.h>
+#include <zephyr/posix/signal.h>
 #else /* defined(CONFIG_POSIX_API) */
 #include <zephyr/net/net_ip.h>
 #include <zephyr/net/socket.h>
 #endif /* defined(CONFIG_POSIX_API) */
 #include <zephyr/shell/shell.h>
 #endif /* defined(CONFIG_ZEPHYR) */
 
+#define signal(a, b) (void)(b)
```

Comment:
should this be for Zephyr only? or at least `#ifndef signal`

Reply:
I'll add ifndef

Reply:
done